applications/nrf_desktop/description.rst

```diff
@@ -1044,9 +1044,33 @@ See the following list of possible scenarios and best practices:
 * Radio frequency (RF) noise can negatively affect the HID report rate for wireless connections.
   If a HID report fails to be delivered in a given Bluetooth LE LLPM connection event, it is retransmitted in the subsequent connection event, which effectively reduces the report rate.
   By avoiding congested RF channels, the :ref:`nrf_desktop_ble_qos` helps to achieve better connection quality and a higher report rate.
-* For the USB device connected directly, you can configure your preferred USB HID poll interval using the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option.
-  By default, the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option is set to ``1`` to request the lowest possible poll interval.
-  Set parameters are not enforced, meaning that the HID host may still eventually use a value greater than the USB polling interval requested by a peripheral.
+* For the USB device connected directly, the applicable options will vary depending on the used USB stack:
+
+  * If you use the USB legacy stack, you can configure your preferred USB HID poll interval using the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option.
+    By default, the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option is set to ``1`` to request the lowest possible poll interval.
+  * If you use the USB next stack, you can configure your preferred USB HID polling rate using the ``in-polling-rate`` property of a DTS node compatible with ``zephyr,hid-device``.
+    The lowest polling rate that is supported by the USB High-Speed is 125 us, which corresponds to an 8 kHz report rate.
+    The lowest polling rate supported by devices that do not support USB High-Speed is 1000 us, which corresponds to 1 kHz report rate.
```

Comment:
```suggestion
    The lowest polling rate that is supported by the USB High-Speed is 125 µs, which corresponds to 8 kHz report rate.
    The lowest polling rate supported by devices that do not support USB High-Speed is 1000 µs, which corresponds to 1 kHz report rate.
```

Reply:
Though from what I've found both can be used

Reply:
Would you like to change these as well @alstrzebonski?

Reply:
I've changed it already

Reply:
Oh sorry, I've missed that it's in two places.